mozilla/fx-private-relay
# MPP-3124 No Favicon on bookmark bar when bookmarking Firefox Relay site

This PR fixes MPP-3124 and #3431 
 
# New feature description

Fix for error we get about not finding our favicon: 

<img width="1908" alt="image" src="https://github.com/mozilla/fx-private-relay/assets/3924990/c16aeeee-8ae7-40bc-a6ec-71dca01a483c">

# Screenshot (if applicable)

<img width="583" alt="image" src="https://github.com/mozilla/fx-private-relay/assets/3924990/8fcaf6d1-a53b-4ee5-990b-5a2e8f66e4ce">

# How to test

Review the console for any errors related to the favicon. 


1. Launch the latest version of the Firefox web browser.
2. Clear the browser cache and ensure there are no existing bookmarks for the URL: 127.0.0.1:8000/accounts/profile/
3. Navigate to the URL: 127.0.0.1:8000/accounts/profile/.
4. Verify that the webpage loads correctly and the favicon is displayed correctly in the tab of the browser.
5. Right-click on the bookmark bar and select "Bookmark This Page" to add it to the bookmarks.
6. Locate the bookmark for the URL 127.0.0.1:8000/accounts/profile/ on the bookmark bar.
7. Verify that the bookmark icon displays the correct favicon image, and not the default "no favicon" image. 

# Checklist (Definition of Done)
